app/controllers/images_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ImagesController < ApplicationController
+  def new; end
+
+  def create
+    @image = Image.new(image_params)
+    if @image.save
+      redirect_to image_path(@image)
+    else
+      flash.now[:danger] = 'Invalid URL'
+      render :new
```

Comment:
When it returns `:new` as a response, this will be sent as a "successful" response (ie, status 200). Let's add a `status` argument to indicate that this was a failure. Try `render :new, status: :unprocessable_entity`